### DISCUSSION TITLE
Stuck in the "Queued" status

### DISCUSSION THREAD

**Discussion by badawi1713** [vercel/community#4363] [Bug Report] [answered]
### Summary

Please, I need help with my deployment because it has been stuck in the 'Queued' status since last night, and there have been no failed builds before.

### Steps to Reproduce

<img width="1463" alt="image" src="https://github.com/vercel/community/assets/44182792/c5fe0ca8-1429-47ea-8cbe-d666921290bd">

**Comment by amyegan** [comment]
Hi @badawi1713! I just checked on your project and it looks like you had a deployment succeed today. Are you still experiencing this problem, or do things seem to be back to normal for you?

**Reply by badawi1713** [reply]
Thanks for the follow-up, it happens to be back to normal again.

**Comment by bethany-paddl** [comment]
I'm having a similar issue, the deployments are all stuck in the queued state. Hopefully like in @badawi1713 's case it will resolve on it's own, but it's a little stressful waiting and not knowing.

**Reply by bethany-paddl** [reply]
Yep, resolved by itself a few hours later.

**Comment by aryan-mehta05** [comment]
I'm facing the same issue today, but according to previous discussions, the following website shows a status of **Degraded Performance**.
They provide real-time updates on Vercel's status.

[https://www.vercel-status.com/](https://www.vercel-status.com/)

**Comment by amyegan** [comment, marked as answer]
Thanks for reporting this! The related incident is now resolved. All queued builds have now been processed and completed.

Incident Report: https://www.vercel-status.com/incidents/6zzfgq5m2jqs
